data/tlg0627/tlg001/__cts__.xml

```diff
@@ -1,7 +1,7 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0627" xml:lang="grc" urn="urn:cts:greekLit:tlg0627.tlg001">
-  <ti:title xml:lang="eng">De prisca medicina</ti:title>
+  <ti:title xml:lang="lat">De prisca medicina</ti:title>
   <ti:edition urn="urn:cts:greekLit:tlg0627.tlg001.1st1K-grc1" workUrn="urn:cts:greekLit:tlg0627.tlg001">
```

Comment:
the edition needs a lang attribute `xml:lang="grc"`

Reply:
O.k I'll admit I don't understand what is being asked here. None of the edition lines in the other CTS files have a lang attribute. And I checked a number of cts_.xml files for other works and they don't seem too either.

Reply:
Maybe not in OGL but it's standard in Perseus. Each edition and translation requires a lang attribute.

Reply:
Interesting. I'm pretty sure its not in most of the cts_.xml files for OGL works, might be somethings we want to globally change to be standard with Perseus, since I think it makes sense to have a language tag for editions.

Reply:
I think that the Captains standards changed (the absence of the attribute used to cause tests to fail), but it was one of the first things I was trained on, so I keep it. 
Technically, I think only translation and commentary require it now, but it just is easier to have it everywhere because it ensures you don't forget it elsewhere.

Reply:
I should add I probably should just follow the OGL on this — force of habit. So I withdraw my comment on it.